source/solution_deploy/lib/__snapshots__/member-stack.test.ts.snap

```diff
@@ -507,7 +507,7 @@ exports[`member stack snapshot matches 1`] = `
       },
       "Type": "AWS::SSM::Parameter",
     },
-    "SHARRRemediationKeyAlias5531874D": {
+    "SHARRRemediationKeyAliasF7D1FD7E": {
```

Comment:
Uh oh, I had to break out the alias so I could keep the logical ID the same, looks like that didn't even work. I'll override it.